src/herder/Upgrades.cpp

```diff
@@ -1388,14 +1388,34 @@ ConfigUpgradeSetFrame::upgradeNeeded(LedgerSnapshot const& ls) const
 }
 
 void
-ConfigUpgradeSetFrame::applyTo(AbstractLedgerTxn& ltx) const
+ConfigUpgradeSetFrame::applyTo(AbstractLedgerTxn& ltx, Application& app) const
```

Comment:
Nit: We can just use `Config::CURRENT_LEDGER_PROTOCOL_VERSION` and avoid having to pass in app.

Reply:
Tests can update this value. Maybe that doesn't that matter here, but Dima's PR also passes in app so we might as well use it.

Reply:
FWIW this is a static const member, so tests can't update it and we should really only access it via class. My PR actually fixes some cases where we unnecessarily plumb the config just to get it. But my PR indeed passes App here  (could be just LedgerManager though), so I'm ok with the interface. But I think we should still access the constant via class. I can address this in my PR during merge.
I think we just need to make this a free constant, as C++ makes this unnecessarily confusing.

Reply:
Ah my bad. I mixed this up with a different config setting.